### PR TITLE
Add command line argument for logging tags

### PR DESCRIPTION
**Pull request by basine**
We would like to log tags that we log through ZMLog via command line. This should enable us to add a comma separated list of tags to log.

### CODE REVIEW

File: WireExtensionComponents/Utilities/AutomationHelper.swift [hunk 1]
```diff
@@ -85,6 +89,13 @@ import zmessaging
         return ZMEmailCredentials(email: email, password: password)
     }
     
+    // Switches on all flags that you would like to log listed after `--debug-log=` tags should be separated by comma
+    fileprivate static func enableLogTags(_ arguments: ArgumentsType) {
+        guard let tagsString = arguments.flagValueIfPresent(AutomationKey.LogTags.rawValue) else { return }
+        let tags = tagsString.replacingOccurrences(of: " ", with: "").components(separatedBy: ",")
```
Comment (daehn) [inline WireExtensionComponents/Utilities/AutomationHelper.swift]: Some tags might have spaces in them

Comment (basine) [inline WireExtensionComponents/Utilities/AutomationHelper.swift]: too late! :P Will make another PR
